libfuse/fs.go

```diff
@@ -354,6 +354,12 @@ func (f *FS) Statfs(ctx context.Context, req *fuse.StatfsRequest, resp *fuse.Sta
 		Namelen: ^uint32(0),
 		Frsize:  fuseBlockSize,
 	}
+	if _, err := libkbfs.GetCurrentSessionIfPossible(
+		ctx, f.config.KBPKI(), false); err != nil {
```

Comment:
Ah.  I think you should use `true` instead, to keep from suppressing unexpected errors.